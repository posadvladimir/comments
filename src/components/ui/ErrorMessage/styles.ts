
import { styled } from 'styled-components';

export const ErrorWrapper = styled.div`
    color: red;
    border: 2px solid;
    border-radius: 5px;
    padding: 10px;
    font-weight: 700;
    text-align: center;
    background: #523030;
    margin-top: 30px;
`;