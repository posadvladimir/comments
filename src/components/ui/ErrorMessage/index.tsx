
import { ERROR_TEST_ID } from 'src/constants/test-ids';

import * as S from './styles';

const ErrorMessage: React.FC = () => (
    <S.ErrorWrapper data-testid={ERROR_TEST_ID}>Произошла ошибка при загрузке данных</S.ErrorWrapper>
);

export default ErrorMessage;