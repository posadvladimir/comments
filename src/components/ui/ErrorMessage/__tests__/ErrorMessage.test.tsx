
import { runSnapshotTest } from 'src/common/tests/snapshot';

import ErrorMessage from '..';

describe('ErrorMessage', () => {
    runSnapshotTest(<ErrorMessage />);
});