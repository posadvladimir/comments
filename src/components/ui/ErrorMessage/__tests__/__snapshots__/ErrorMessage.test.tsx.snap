// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage соотвествует снепшоту 1`] = `
<div
  className="sc-bcPKhP gFqXVc"
  data-testid="error-test-id"
>
  Произошла ошибка при загрузке данных
</div>
`;
